scripts/enrich_rulings_ai.py

```
@@ -87,13 +85,13 @@ def ai_extract_q_and_a(raw_text: str) -> dict[str, str] | None:
 def ai_generate_tags(ruling_text: str, existing_tags: list[str]) -> list[str]:
     """
     Generate a list of relevant tags for a ruling based on its text content and existing tags.
-    
+
     Adds tags such as "timing_window" or "cancellation_effect" if corresponding keywords are detected in the ruling text, merges them with any existing tags, and returns a sorted list.
```

Ruff (0.11.9)

Line too long (184 > 100)

(E501)